packages/main/src/Calendar.js

```diff
@@ -143,52 +164,69 @@ const metadata = {
 /**
  * @class
  *
+ * <h3 class="comment-api-title">Overview</h3>
+ *
+ * The <code>ui5-calendar</code> can be used stand alone to display the years, months, weeks and days
+ * but the main purpose of the <code>ui5-calendar</code> is to be used within a <code>ui5-date-picker</code>
+ * <br><br>
+ *
+ * <h3>Usage</h3>
+ *
+ * The user can navigate to a particular date by:
+ * <br>
+ * <ul>
+ * <li>Pressing over a month inside the <code>ui5-monthpicker</code> component</li>
```

Comment:
ui5-month-picker ui5-year-picker ui5-day-picker are internal components and the end users do not have the context, use more user friendly explanation - selecting an year from the year view, selecting a month from the month view, picking a date from the days/weeks view

Reply:
Done